src/Components/ADT3DSceneBuilder/Internal/Behaviors/Internal/VisualRulesTab.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { VisualRulesList } from '../VisualRules/VisualRuleList';
+
+export const VisualRulesTab: React.FC = () => {
```

Comment:
aren't there some common things all the tabs have at the root when it comes to styles or something?

Reply:
yeah, left panel styles, but I don't have any elements to attach styles to atm.